azure-pipelines.yml

```diff
@@ -122,8 +122,18 @@ stages:
     pool:
       vmImage: 'ubuntu-latest'
     jobs:
-      - job: build_push_test_
-        timeoutInMinutes: 120
+      - job: make_suffix
```

Comment:
I've added a new `make_suffix` job so we can share this suffix among each docker image (there is 1 docker image per python version).

Reply:
❤️